Vagrantfile

```diff
@@ -0,0 +1,46 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Install [vagrant](https://www.vagrantup.com/downloads.html)
+# Install [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+# common commands:
+# vagrant up    
+# vagrant ssh    
+#     cd /src
+#     bazel build //sfdb:sfdb
+# vagrant halt    
+# vagrant destroy 
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/bionic64"
+  config.vm.box_version = "= 20200325.0.0"
+  config.vm.synced_folder ".", "/src"
+  config.vm.network "forwarded_port", guest: 12987, host: 8000, host_ip: "127.0.0.1"
+
+  # Work around disconnected virtual network cable.
+  config.vm.provider "virtualbox" do |vb|
+    vb.name = "sfdb_build"
+
+    # Configure resources dedicated to the VM
+    vb.memory = "6144"
```

Comment:
is this a minimum or optimal value?

Reply:
It seems to be around the minimum required. CPU and memory at ~90%